devtools/gradle/src/functionalTest/resources/add-extension-multi-module-kotlin-dsl/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+buildscript {
+
+    repositories {
+        jcenter()
+        mavenLocal()
+        mavenCentral()
+        gradlePluginPortal()
+    }
+
+}
+
+plugins {
+    java
+}
+
+group = "com.quarkus.demo"
+version = "1.0"
+
+
+subprojects {
+
+    apply(plugin = "java")
+
+    java {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
I think you should remove this, since our CI will be also testing it under Java 11. And I remember we had issues with Kotlin and Java 14. If you run into that, the tests may need to be disabled for that version.

Reply:
See https://github.com/quarkusio/quarkus/blob/master/integration-tests/gradle/build.gradle#L42